Dockerfile

```diff
@@ -2,11 +2,15 @@
 FROM python:3.9
 
 # Set working directory.
+WORKDIR exp/
 
 # Expose port 8000 for your FastAPI server.
-
-# Install Python requirements using Pip.
+EXPOSE 8000
 
 # Copy your codebase into the Docker container.
+COPY . .
+# Install Python requirements using Pip.
+RUN pip install -r requirements.txt
 
 # Run FastAPI server on the port exposed above.
+CMD ["uvicorn","main:app","--reload"]
```

Comment:
You need to specify the host address here ([reason](https://stackoverflow.com/a/20778887)).